backtester/config/examples/ftx-cash-carry.strat

```diff
@@ -58,8 +59,9 @@
    },
    "min-slippage-percent": "0",
    "max-slippage-percent": "0",
+   "maker-fee-override": "0.001",
```

Comment:
This is missing taker-fee-override. Both these either need to be the same or removed because it skews the fees when it fetches the taker fee rate.